src/common/entities/base.entity.ts

```diff
@@ -0,0 +1,9 @@
+import { CreateDateColumn, PrimaryGeneratedColumn } from 'typeorm';
+
+export class BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @CreateDateColumn({ type: 'timestamp', name: 'created_at' })
+  createdAt: Date;
```

Comment:
WDYT about adding the updatedAt attribute as well? And also excluding it and createdAt from being sent frontend, you can use class-transformer

Something like:
```
@UpdateDateColumn()
  @Exclude({ toPlainOnly: true })
  updatedAt?: Date;
```

Reply:
nice, thanks